core/templates/pages/topic-editor-page/editor-tab/topic-editor-stories-list.directive.html

```diff
@@ -7,30 +7,30 @@
   <tr>
     <th ng-repeat="key in STORY_TABLE_COLUMN_HEADINGS"
         class="dashboard-table-headings">
-      <p ng-if="key === 'title'"> Title </p>
-      <p ng-if="key === 'node_count'"> Chapter Count </p>
-      <p ng-if="key === 'publication_status'"> Is Story Published? </p>
+      <p ng-if="key === 'title'" class="text-left">Title</p>
+      <p ng-if="key === 'node_count'" class="text-center">Chapters</p>
+      <p ng-if="key === 'publication_status'" class="text-center">Published?</p>
     </th>
   </tr>
   <tr ng-repeat="summary in storySummaries"
       class="list-item protractor-test-story-list-item">
     <td>
-      <a ng-click="openStoryEditor(summary.getId())" class="list-summary">
+      <a ng-click="openStoryEditor(summary.getId())" class="list-summary text-left">
         <span class="protractor-test-story-title"> <[summary.getTitle()]> </span>
       </a>
     </td>
     <td>
-      <a ng-click="openStoryEditor(summary.getId())" class="list-summary">
+      <a ng-click="openStoryEditor(summary.getId())" class="list-summary text-center">
         <span> <[summary.getNodeTitles().length]> </span>
       </a>
     </td>
     <td>
-      <a ng-click="openStoryEditor(summary.getId())" class="list-summary">
+      <a ng-click="openStoryEditor(summary.getId())" class="list-summary text-center">
         <span class="protractor-test-story-publication-status"> <[summary.isStoryPublished() ? "Yes" : "No"]> </span>
       </a>
     </td>
     <td>
-      <a ng-click="deleteCanonicalStory(summary.getId())" class="list-summary">
+      <a ng-click="deleteCanonicalStory(summary.getId())" class="list-summary text-center">
```

Comment:
Even after I deleted the story, it was still shown locally until I refreshed the page. Can you take a look at that?

Reply:
Hey @aks681 I am unable to repro this, can you check once more?(Gif attached.)
![story delete](https://user-images.githubusercontent.com/30312043/85517928-5e435800-b61d-11ea-82b5-12d5fb23acd2.gif)

Reply:
Hmm, this does happen for me still. Try creating 2 stories and deleting one of them? That's when I encountered it.

Reply:
Yeah fixed this, was happening becaues the code was trying to access .id whereas the property was _id. so changes that to getId() and now it's fine.